src/Symfony/Component/Console/Helper/ProgressBar.php

```diff
@@ -486,6 +486,14 @@ private function overwrite(string $message): void
                 if ($this->output instanceof ConsoleSectionOutput) {
                     $messageLines = explode("\n", $this->previousMessage);
                     $lineCount = \count($messageLines);
+
+                    $lastLineWithoutDecoration = Helper::removeDecoration($this->output->getFormatter(), end($messageLines) ?? '');
+
+                    // When the last previous line is empty (without formatting) it is already cleared by the section output, so we don't need to clear it again
+                    if (empty($lastLineWithoutDecoration)) {
```

Comment:
We tend to avoid using `empty()` in Symfony.

Reply:
Fixed, thanks